packages/open-next/src/overrides/imageLoader/dummy.ts

```diff
@@ -0,0 +1,11 @@
+import { ImageLoader } from "types/open-next";
```

Comment:
Use `import type` - it applies to all the dummy overrides.
I think there is an autofix rule for eslint.

Reply:
I think it's https://typescript-eslint.io/rules/consistent-type-imports/

Reply:
Nice catch, i'll add it later today in this PR as well as fixing all the missing one